packages/slate-editor/src/modules/nodes-hierarchy/normilizers/isInside.ts

```diff
@@ -0,0 +1,19 @@
+import { Node } from 'slate';
+
+import type { HierarchyFixer, HierarchyNormalizer, HierarchyNodeQuery } from '../types';
+
+export function isInside(
```

Comment:
A part of me doesn't like this name. I guess it's just too different from the Slate terminology. 

I'd say a function that checks if the path is inside a given node type it should be named `isDescendantOf()` or something like that.

Reply:
Agree, will rename